Catalog apis release note and upcoming changes update (#312)
eu-cdse/documentation: APIs/ReleaseNotes.qmd

This page will include the list of changes made to Catalog APIs for each release.

> # OData Catalog API updates

Reviewer: ```suggestion
## OData Catalog API updates
```

> ## [2023-10-02] OData API single product response update

Reviewer: ```suggestion
### OData API single product response update [2023-10-02]
```

> To be compliant with the OData PRIP interfaces the **OData API** response for a single product query was updated. The response now returns the product itself instead of a list containing the product (assigned to the field `value`). The example reflecting the change is presented below.

Reviewer: ```suggestion
To comply with the OData PRIP interfaces, the **OData API** response for a single product query was updated. The response returns the product itself instead of a list containing the product (assigned to the field `value`). The example reflecting the change is presented below:
```
